modules/log6x-logging-loki-cli-install.adoc

```diff
@@ -0,0 +1,209 @@
+//
+// * observability/logging/log_storage/installing-log-storage.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-logging-loki-cli-install_{context}"]
+= Installing the {loki-op} by using the CLI
+
+To install and configure logging on your {product-title} cluster, you must first install {loki-op}, which manages the log store `Loki`. `Loki` log store is deployed and configured by the {loki-op} by reconciling the resource `LokiStack`. This can be done from the {product-title} command-line interface (CLI).
+
+.Prerequisites
+
+* You have administrator permissions.
+* You installed the {oc-first}.
+* You have access to a supported object store. For example: AWS S3, Google Cloud Storage, Azure, Swift, Minio, or {rh-storage}.
+
+.Procedure
+
+. Create a `Namespace` object for {loki-op}:
++
+.Example `Namespace` object
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-operators-redhat # <1>
+  labels:
+    openshift.io/cluster-monitoring: "true" # <2>
+----
+<1> You must specify the `openshift-operators-redhat` namespace. To enable monitoring for the operator, you should configure the Cluster Monitoring Operator to scrape metrics from the `openshift-operators-redhat` namespace and not the `openshift-operators` namespace. The `openshift-operators` namespace might contain community Operators, which are untrusted and could publish a metric with the same name as an {product-title} metric, which would cause conflicts.
+<2> A string value that specifies the label as shown to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
+
+. Apply the `Namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a `Subscription` object for {loki-op}:
++
+.Example `Subscription` object
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: loki-operator
+  namespace: openshift-operators-redhat # <1>
+spec:
+  channel: stable-6.<y> # <2>
+  installPlanApproval: Automatic # <3>
+  name: loki-operator
+  source: redhat-operators # <4>
+  sourceNamespace: openshift-marketplace
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
+<2> Specify `stable-6.<y>` as the channel.
+<3> If the approval strategy in the subscription is set to `Automatic`, the update process initiates as soon as a new Operator version is available in the selected channel. If the approval strategy is set to `Manual`, you must manually approve pending updates.
+<4> Specify `redhat-operators` as the value. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the `CatalogSource` object that you created when you configured the Operator Lifecycle Manager (OLM).
+
+. Apply the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create an `OperatorGroup` object
```

Comment:
We should create `OperatorGroup` at first, then create the `Subscription`. Or the `Subscription` may raise error.

Reply:
done